### DISCUSSION TITLE
How to execute gdb commands in relp

### DISCUSSION THREAD

**Discussion by zul00** [mfussenegger/nvim-dap#432] [Q&A] [answered]
Sorry, maybe I missed this but is it possible to execute default gdb command like `info`, `breakpoints` inside of REPL?

Right now I can only execute the one that is available from nvim-dap.

I am using 'vscode-cpptools'

**Comment by theHamsta** [comment, marked as answer]
As far as I know vscode-cpp does not expose the gdb commands directly (at least the last time that I checked in summer)

**Reply by zul00** [reply]
Thank you, now I know what I am looking for

I found it here, that you can prepend the command with `-exec` to execute native command

https://code.visualstudio.com/docs/cpp/cpp-debug#_gdb-lldb-and-lldbmi-commands-gdblldb
